### ISSUE TITLE
[BUG] Pro controller issues

### ISSUE THREAD

**Issue by APerson976** [Davidobot/BetterJoyForCemu#1186]
So I've been trying to use betterjoy to use gyro in citra. But, when the program detects my controller it spams home and capture and spams the low battery notification. It also says duplicate timestamp enqueued.

Expected behavior:
I expected the program to function as it should.

 - Type: Pro controller
 - Connection: Bluetooth

**Comment by TheAvacad0** [comment]
Yeah it’s just gone to shit I’ve tried everything and this app hasn’t been updated since 2020
